src/main.rs

```diff
@@ -352,6 +352,13 @@ fn run() -> Result<(), failure::Error> {
                     "{} Start a local server for developing your worker",
                     emoji::EAR
                 ))
+                .arg(
+                    Arg::with_name("env")
+                        .help("environment to build")
+                        .short("e")
+                        .long("env")
+                        .takes_value(true)
+                )
```

Comment:
i already had the `.matches("env")` check and target creation logic, just needed to add the clap argument :)